datameta/views/setpass.py

```diff
@@ -1,107 +0,0 @@
-# Copyright (c) 2021 Universität Tübingen, Germany
```

Comment:
Please restore this file, this is the `/setpass` view for use with recovery tokens 😅 . Let's refactor that view to the new API endpoint in a separate PR.

Reply:
Done